gocyclo.go

```diff
@@ -98,6 +101,14 @@ func isDir(filename string) bool {
 	return err == nil && fi.IsDir()
 }
 
+func isIgnored(path string) bool {
+	if len(*ignore) == 0 {
+		return false
+	}
+	match, err := regexp.MatchString(*ignore, path)
```

Comment:
Actually, you should probably implement the (`flag.Value`)[https://godoc.org/flag#Value] interface for a regexp while you're here.